system/doc/reference_manual/macros.xml

```diff
@@ -255,9 +255,11 @@ or
     %% Code that will work in OTP 22 or higher
   -elif(?OTP_RELEASE >= 21).
     %% Code that will work in OTP 21 or higher
+  -else.
+    %% OTP 20 or lower.
   -endif.
 -else.
-  %% OTP 20 or lower.
+  %% OTP_RELEASE is not defined.
```

Comment:
This change is wrong. The `-else.` _implies_ that `OTP_RELASE` isn't defined, but the _implication_ is that we're on OTP-20 or older, which is the information we want.